components/ILIAS/ResourceStorage/classes/Container/View/Configuration.php

```diff
@@ -45,6 +48,7 @@ public function __construct(
         global $DIC;
         $this->ctrl = $DIC->ctrl();
         $this->action_provider = new ExternalActionProvider();
+        $this->top_action_provider = new ExternalActionProvider();
```

Comment:
FMPOV we don't need an additional `ExternalActionProvider` here, I will open a PR against your branch